lib/alex_koin/slack_commands.ex

```diff
@@ -104,4 +103,11 @@ defmodule AlexKoin.SlackCommands do
 
     Repo.all(Wallet.by_minimum_balance(min_balance))
   end
+
+  def fact() do
+    fact_funcs = AlexKoin.Factoids.__info__(:functions)
+
+    {func_name, _arity} = Enum.random(fact_funcs)
+    apply(AlexKoin.Factoids, func_name, [])
```

Comment:
alias `Factoids` please. As soon as there's more than one I like to do so. Just creates a good convention for growth